src/components/ScriptureWorkspaceCard.js

```diff
@@ -138,48 +138,7 @@ ScriptureWorkspaceCard.propTypes = {
 }
 
 
-
-/* code graveyard
-
-      {
-        data.perf
-        ?
-        <div className="text-sm max-w-prose">
-          <pre>{JSON.stringify(data.perf,null,4)}</pre>
-        </div>
-        :
-        <CircularProgress/>
-
-      }
-*/
 /*
+            editable={data.url ? true : (id.endsWith(owner) ? true : false)}
```

Comment:
Don't you want to remove all the changes in this file?

Reply:
removed the comment